tests/unit/helpers.js

```diff
@@ -1470,3 +1474,68 @@ export const createExperimentData = ({
 }) => {
   return { [id]: variantId };
 };
+
+const queryByClassName = (container, className) => {
+  return container.querySelectorAll(`.${className}`);
+};
+
+const getByClassName = (container, className) => {
+  const elements = queryByClassName(container, className);
+  if (!elements.length) {
+    return null;
+  }
```

Comment:
in the testing library [api](https://testing-library.com/docs/queries/about#types-of-queries), `get` would raise for 0 matches.  - it'd be `queryBy` to get `null` for empty.  I.e. this function should be called `queryByClassName` to be consistent (though that conflicts with the raw function you already have with that name).

Reply:
I agree that we should be more consistent with these, and `getBy` should raise for 0 matches. I will update these to make them work that way.